storybook/src/components/ImageSlider/ImageSlider.stories.tsx

```diff
@@ -14,27 +14,27 @@ const meta = {
     images: [
       {
         alt: 'Bridge',
-        aspectRatio: 'x-wide',
+        aspectRatio: '16:9',
```

Comment:
This is the default aspect ratio for the Image component – all of these can be removed.